pkgs/sage-conf/pyproject.toml.m4

```diff
@@ -0,0 +1,45 @@
+include(`sage_spkg_versions_toml.m4')dnl' -*- conf-toml -*-
+[build-system]
+requires = [
+    SPKG_INSTALL_REQUIRES_setuptools
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "sage-conf"
+description = "Sage: Open Source Mathematics Software: Configuration module for the SageMath library"
+readme = "README.rst"
+dnl Not including the standard metadata from pyproject_toml_metadata.m4
+dnl because sage-conf is GPL v3+.
+license = {text = "GNU General Public License (GPL) v3 or later"}
+authors = [{name = "The Sage Developers", email = "sage-support@googlegroups.com"}]
+classifiers = [
+    "Development Status :: 6 - Mature",
+    "Intended Audience :: Education",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
+    "Operating System :: POSIX",
+    "Operating System :: MacOS :: MacOS X",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Scientific/Engineering :: Mathematics",
+]
+urls = {Homepage = "https://www.sagemath.org"}
+requires-python = ">=3.9, <3.13"
```

Comment:
Can we get rid of the upper bound here? This package is relatively simple python that should not be tied to the version of python.

This allows more flexibility (e.g. to use a released sage-conf with a beta version of sagemath-standard when one is experimenting with an update of python).

Reply:
No. The version bound is set for uniformity.